### DISCUSSION TITLE
React Native - Property 'atob' doesn't exist

### DISCUSSION THREAD

**Discussion by issamahwach** [WalletConnect/walletconnect-monorepo#4425] [Developer Support]
Following the steps in docs for integrating Web3Modal with WagmiV1 on Expo is showing the following error:
` ERROR  ReferenceError: Property 'atob' doesn't exist, js engine: hermes
 ERROR  Invariant Violation: "main" has not been registered. This can happen if:
* Metro (the local dev server) is run from the wrong folder. Check if Metro is running, stop it and restart it in the current project.
* A module failed to load due to an error and `AppRegistry.registerComponent` wasn't called., js engine: hermes`

"expo": "~50.0.14"
"react": "18.2.0"
"react-native": "0.73.6"
"wagmi": "1.4.13"
"viem": "1.21.4"
"@web3modal/wagmi-react-native": "^1.3.0"

**Comment by ignaciosantise** [comment]
hey @issamahwach 👋 can you provide a public repository with the issue?
